packages/x-components/tailwind.config.js

```diff
@@ -3,13 +3,27 @@ const options = require('./src/tailwind/plugin-options');
 
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ['./public/index.html', './src/**/*.vue'],
+  content: [
+    './public/index.html',
+    './src/**/*.vue',
+    './node_modules/@empathyco/x-tailwindcss/showcase/**/*.js'
```

Comment:
We have to be very careful with this if moved to the archetype. With this line it will include the full set of components CSS. It should only be included for development purposes.

Reply:
This is another proposal that I thought yesterday (yeah, I know, too late) about exposing a dev server rather than a set of Vue components.
In my mind this would allow us to remove the rollup config for the showcase components, it would facilitate using this from a custom project regardless the components library used, and also ease up this configuration (run a command instead of setting up a view and a development tailwind config).
https://searchbroker.atlassian.net/browse/EX-7329

Reply:
I love how that sounds Javi :sisi3: